authors/apps/profiles/urls.py

```diff
@@ -1,14 +1,21 @@
 from django.urls import path
+
 from .views import (
     UserProfileView,
     UserProfileUpdateView,
     UserProfileListView,
+    MyFollowersAPIView,
+    UnfollowDestroyAPIView,
+    FollowCreateAPIView,
+    MyFollowUpsAPIView,
 )
 
-
 urlpatterns = [
     path('<username>', UserProfileView.as_view(), name='profile-detail'),
     path('<username>/edit', UserProfileUpdateView.as_view(), name='profile-update'),
     path('', UserProfileListView.as_view(), name='profile-list'),
-
+    path('myfollowup/', MyFollowUpsAPIView.as_view(), name='get_i_follow'),
```

Comment:
@Manueldominic, please use the endpoints I requested .
These conflict with the one above